provider/views.py

```diff
@@ -255,7 +327,36 @@ def error_response(self, request, error, **kwargs):
 
         return self.render_to_response(ctx, **kwargs)
 
+    def get_implicit_response(self, request, client):
+        """
+        Assumes validated and authorized request with a response_type that is
+        associated with an implicit flow.
+        """
+        data = self.get_data(request)
+
+        lookup_kwargs = {
+            "user": request.user,
+            "client": client,
+            "scope": scope.to_int(*data.get('scope', '').split())
+        }
+
+        if constants.SINGLE_ACCESS_TOKEN:
+            token = self.get_access_token(request, **lookup_kwargs)
+        else:
+            token = self.create_access_token(request, **lookup_kwargs)
+
+        response_data = self.access_token_response_data(token, data['response_type'])
+
+        basepath = data.get("redirect_uri")
+        if not basepath:
```

Comment:
@justinabrahms thanks for calling that out.  I hadn't (intentionally) modified the validation, which I believe was already handled [here](https://github.com/edx/django-oauth2-provider/blob/jsa/implicit2/provider/oauth2/forms.py#L162).  Will double check for loopholes.